src/Tools/dotnet-monitor/Startup.cs

```diff
@@ -46,7 +46,6 @@ public void ConfigureServices(IServiceCollection services)
                 // Allow serialization of enum values into strings rather than numbers.
                 options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
             })
-            .SetCompatibilityVersion(CompatibilityVersion.Latest)
```

Comment:
API was deprecated and generated a build warning-as-error. This API actually didn't do anything since it is a no-op since ASP.NET Core 3.0; probably a relic from when the project started as a ASP.NET Core 2.1 application.